integration/wai-aria.spec.js

```diff
@@ -39,240 +54,272 @@ describe('WAI ARIA Spec', () => {
             expect(title).toBe(
                 'React Accessible Accordion - Integration Test Sandbox',
             );
-
-            await browser.close();
```

Comment:
See above - I now do this on `afterEach` instead (prevents memory leaks when tests fail).